setup.py

```diff
@@ -179,6 +179,8 @@ def package_files(prefix: str, directory: str, extension: str):
     'datasets>=2.4,<3',
 ]
 
+extra_deps['sentencepiece'] = ['sentencepiece==0.1.97']
```

Comment:
[nit] Do we need to create another build target?  Can we just include this in the `nlp` dep?

Reply:
I'd like to keep it separate because I've had issues installing the `sentencepiece` dependency in the past, so I don't want to force all `nlp` users to install it. Sound ok?

Reply:
Could you elaborate on what type of issues a bit more?

I think it's ok to keep it as it's own dependency for now then in light of that.

Reply:
I had to manually install protobuf when I was testing this out on windows subsystem for linux. I don't really know if this is a persistent issue, as I did not encounter it on my mac, or in CI.